packages/ui/src/2_molecules/Table/components/TableDesktop/components/TableRow/TableRow.module.css

```diff
@@ -0,0 +1,45 @@
+.row {
```

Comment:
This is almost the exact copy of styles from `TableBase/TableRow`. Ask Gilad if it's fine to use the same styling and if so, have just one config in some shared location. If not, I'd still unify it somehow.